env/build_singularity_gnu.env

```diff
@@ -0,0 +1,16 @@
+#Setup instructions for singularity container using gnu 9.3.0
+
+source /usr/share/lmod/6.6/init/profile
+
+module purge
+
+module use /opt/hpc-modules/modulefiles/stack
+
+module load hpc hpc-gnu hpc-openmpi
+
+module load netcdf hdf5 bacio sfcio sigio nemsio w3emc esmf fms crtm g2 png zlib g2tmpl ip sp w3nco cmake/3.21.1 gfsio wgrib2 upp
```

Comment:
Is there a reason this is not doing a `module load srw_common` like the rest of the env files?

Reply:
@christopherwharrop-noaa We are trying to build the containers: still frequent update. I think we can adopt srw_common  soon on singularity but it needs time. @EdwardSnyder-NOAA can you add a comment line about the container url in the singularity env file? just for reference but details will be available at docs.